### PR TITLE
applied suggestion

### PR DESCRIPTION
**Pull request by remidhum**
Noticable changes: Handson now in two words, Web site in one, Updated ATI to AMD
Minor changes: Changed to consistant American English spelling, adjusted various pronouns and checked -ing endings

### CODE REVIEW

**Comment by remidhum** [comment]
:

**Comment by remidhum** [comment]
:+1:

File: guide/controller-plugin.md [hunk 1]
```diff
@@ -210,9 +210,9 @@ motor position range are written in red.
 
 ### Remote-control Plugin
 
-A remote-control plugin allow to simply and efficiently create an interface
+A remote-control plugin allows to simply and efficiently create an interface
 using the Webots API to communicate with a real robot. The main purpose of a
-remote-control library is to wrap all the Webots API functions used by the robot
+remote-controled library is to wrap all the Webots API functions used by the robot
```
Comment (omichel) [inline guide/controller-plugin.md]: I believe this name should remain remote-control library, as "remote-control" is the name of the library.

File: guide/controller-plugin.md [hunk 2]
```diff
@@ -294,7 +294,7 @@ For example, if you want to be able to use the distance sensor of the real
 robot, you have to wrap the `wbr_set_refresh_rate()` function (to set the
 internal state of the remote control library to read this distance sensor only
 when required), and to call `wbr_distance_sensor_set_value()` into the
-remote-control library when the distance sensor is refreshed (typically into the
+remote-controled library when the distance sensor is refreshed (typically into the
```
Comment (omichel) [inline guide/controller-plugin.md]: Same comment here.

File: guide/cpp-java-python.md [hunk 3]
```diff
@@ -54,8 +54,8 @@ controller, and vice versa. For example it won't be possible to call
 %end
 
 Generally, the user-defined controller class should have a `run()` function that
-implements the main controller loop. That loop should contains a call to the
-`Robot`'s `step()` method. Then the only responsibility of the controller's
+implements the main controller loop. That loop should contain a call to the
+`Robot`'s `step()` method. Then, the only responsability of the controller's
```
Comment (omichel) [inline guide/cpp-java-python.md]: Not sure about his one: https://en.wiktionary.org/wiki/responsability

File: guide/introduction.md [hunk 4]
```diff
@@ -2,10 +2,9 @@
 
 Webots can execute controllers written in compiled (C/C++, Java) or interpreted
 (Python, MATLAB) languages. The compilation or interpretation process requires
-extra software that must usually be installed separately. Only when using C/C++
-on the Windows platform it is not necessary to install a separate C/C++
-compiler; on this platform Webots comes with a pre-installed and preconfigured
-copy of the MinGW C/C++ compiler. For any other language or platform the
+extra software that must usually be installed separately. it is not necessary to
```
Comment (omichel) [inline guide/introduction.md]: Missing capital i for "It is not necessary to".

File: guide/introduction.md [hunk 5]
```diff
@@ -2,10 +2,9 @@
...
+install a separate C/C++ compiler only when using C/C++ on the Windows platform ;
+ on this platform Webots comes with a pre-installed and preconfigured copy of the MinGW C/C++ compiler. For any other language or platform the
```
Comment (omichel) [inline guide/introduction.md]: Extra space at the beginning of this line.

File: guide/linux.md [hunk 6]
```diff
@@ -15,6 +15,6 @@ connection. This problem is not specific to Webots but to most GLX (OpenGL on
 the X Window system) applications that use complex OpenGL graphics. We think
 this is caused by incomplete or defective implementation of the GLX support in
 the graphics drivers on Linux. It may help to run the `ssh -x` tunnel across two
-computers with the same graphics hardware, e.g., both nVidia or both ATI. It
+computers with the same graphics hardware, e.g., both nVidia or both AMD. It
```
Comment (omichel) [inline guide/linux.md]: I believe nVidia should be written all caps: NVIDIA.

File: guide/mac-os-x.md [hunk 7]
```diff
@@ -3,7 +3,7 @@
 ### MATLAB and robot plugins
 
 The controllers and the robot plugins (e.g. the robot windows) are sharing the
-same process. Generally this mechanism is working well, but some instabilities
+same process. Generally this mechanism is works well, but some instabilities
```
Comment (omichel) [inline guide/mac-os-x.md]: We should remove the "is".

File: guide/programming.md [hunk 8]
```diff
@@ -115,7 +115,7 @@ message in question is shown. So this problem often happens when you:
 
 - Have moved the project or source files to a location that does not correspond to
 the above description.
-- Use an external build system, e.g. Visual Studio, that is not configured to
+- Use an external building system, e.g. Visual Studio, that is not configured to
```
Comment (omichel) [inline guide/programming.md]: I believe it's really called a "build system" (computer science terminology).

File: guide/sample-webots-applications.md [hunk 9]
```diff
@@ -1,11 +1,11 @@
 # Sample Webots Applications
 
 This chapter gives an overview of sample worlds provided with the Webots
-package. The examples world can be tried easily; the ".wbt" files are located in
+package. The example worlds can be tested easily; the ".wbt" files are located in
 various "worlds" directories of the "WEBOTS\_HOME/projects" directory and can be
 directly opened from Webots using the `Open Sample World` item in `File` menu.
 The controller code is located in the corresponding "controllers" directory.
-This chapter provides each example a with short abstract only. More detailed
+This chapter provides a only short abstract for each example. More detailed
```
Comment (omichel) [inline guide/sample-webots-applications.md]: It looks strange to me: I would have written "only a short" instead of "a only short"...

File: guide/the-3d-window.md [hunk 10]
```diff
@@ -99,9 +98,9 @@ key and right mouse button together while dragging the mouse. Linux users should
 also hold down the Control key (Ctrl) together with the Alt key. Also, Mac OS X
 users with a one-button mouse should hold down the Control key (Ctrl) to emulate
 the right mouse button. This way your are drawing a 3D-vector with origin the
-center of mass and whose end is located in the plane parallel to the view which
+center of mass and whose end is located on the plane parallel to the view which
 passes through this center. The object is prompted to turn around the vector
-direction, the intensity of the applied torque being directly proportional to
+direction, the intensity of the applied torque directly proportional to
```
Comment (omichel) [inline guide/the-3d-window.md]: Why did you remove "being"? I think it's useful. Maybe we could simplify and replace it with "is".

File: guide/the-user-interface.md [hunk 11]
```diff
@@ -390,7 +388,7 @@ The `License Manager...` item opens the `Webots License Manager` window that
 allows you to see which licenses are in use and optionally to transfer some
 license to your local computer for off-line use. Please note that the transfer
 of license may be limited by your local license administrator: It may not be
-possible to transfer some license to your local computer or only for a limited
+possible to transfer some licenses to your local computer or only for a limited
```
Comment (omichel) [inline guide/the-user-interface.md]: I would keep the singular form as it is what we mean.

File: guide/the-user-interface.md [hunk 12]
```diff
@@ -437,7 +435,7 @@ of HTML pages, PDF documents, etc.
 ### Main toolbar
 
 The main toolbar contains items for editing the world and the speedometer (see
-subsection [this section](#speedometer-and-virtual-time)), other than shortcuts
+ [subsection](#speedometer-and-virtual-time)), other than shortcuts
```
Comment (omichel) [inline guide/the-user-interface.md]: Extra space added here.

Comment (omichel) [inline guide/the-user-interface.md]: Also the sentence seems wrong: "see subsection" &rarr; "see this subsection"

File: guide/transfer-to-your-own-robot.md [hunk 13]
```diff
@@ -80,11 +80,11 @@ system for the e-puck and the Hemisson robot. Samples are located in the
...
-Unlike the remote control system, the cross-compilation system requires that the
-source code of your Webots controller be recompiled using the cross-compilation
-tools specific to your own robot. You will also need to rewrite the Webots
-include files to be specific to your own robot. In simple cases, you can just
-rewrite the Webots include files you need, as in the "hemisson" example. In more
+Unlike the remote control system, the cross-compilation system requires the
+source code of your Webots controller to be recompiled using the cross-compilation
+tools specific to your own robot. You will also need to rewrite the Webots include-files
+to be specific to your own robot. In simple cases, you can just
+rewrite the Webots include-files you need, as in the "hemisson" example. In more
```
Comment (omichel) [inline guide/transfer-to-your-own-robot.md]: I would put a dash between include and files, "include files" is a very common CS terminology.

File: guide/transfer-to-your-own-robot.md [hunk 14]
```diff
@@ -93,12 +93,12 @@ should be uploaded onto the real robot for local execution.
 
 #### Examples
 
-Webots support cross-compilation for several existing commercial robots. For the
+Webots supports cross-compilation for several existing commercial robots. For the
 *e-puck*<sup>TM</sup> robot, this system is fully integrated in Webots and needs
 no modification in the code. For the *Hemisson*<sup>TM</sup> robot, this system
-needs a few include files to replace the Webots API include files. For the
+needs a few include-files to replace the Webots API include-files. For the
```
Comment (omichel) [inline guide/transfer-to-your-own-robot.md]: same comment here

File: guide/transfer-to-your-own-robot.md [hunk 15]
```diff
@@ -93,12 +93,12 @@ should be uploaded onto the real robot for local execution.
...
 *Khepera*<sup>TM</sup> robot, a specific C library is used in addition to
-specific include files.
+specific include-files.
```
Comment (omichel) [inline guide/transfer-to-your-own-robot.md]: Idem.

File: guide/tutorial-4-more-about-controllers-20-minutes.md [hunk 16]
```diff
@@ -97,8 +97,8 @@ in order to be able to use the corresponding API (documented in chapter 3 of the
 
 <!-- -->
 
-> **handson**:
-Just after the include statements add a macro that defines the duration of each
+> **hands on**:
+Just after including the statements add a macro that defines the duration of each
```
Comment (omichel) [inline guide/tutorial-4-more-about-controllers-20-minutes.md]: The include statement refers to the following C programming language statement:
```C
#include <some_file.h>
```
Therefore, the original version should restored.

File: guide/using-the-nao-robot.md [hunk 17]
```diff
@@ -147,8 +147,7 @@ simulation speed drifts away from 1.0x, the physics simulation gets wrong
 (unnatural) and thus Choregraphe motions don't work as expected any more. For
 example if Webots indicates 0.5x, this means that it is only able to simulate at
 half real-time the motion provided by Choregraphe: the physics simulation is too
-slow. Therefore it is important to keep the simulation speed as much as possible
-close to 1.0x. There are currently no means of synchronizing Webots and
+slow. Therefore it is important to keep the simulation speed as clpse as possible to 1.0x. There are currently no means of synchronizing Webots and
```
Comment (omichel) [inline guide/using-the-nao-robot.md]: Typo: clpse %rarr; close

File: guide/using-webots-makefiles.md [hunk 18]
```diff
@@ -78,7 +78,7 @@ Webots C/C++ controllers are regular binary executable files that can easily be
 compiled and linked with external libraries. To add an external library it is
 only necessary to specify the path to the header files, and the path and name of
 the library in the controller's Makefile. For example the `-I`*dir* flag can be
-used to add a directory to search for include files. The LIBRARIES variable can
+used to add a directory to search for include-files. The LIBRARIES variable can
```
Comment (omichel) [inline guide/using-webots-makefiles.md]: See previous comment about include files.

File: guide/verifying-your-graphics-driver-installation.md [hunk 19]
```diff
@@ -2,33 +2,31 @@
...
-site](http://downloadcenter.intel.com). Linux graphics drivers from Intel may be
-obtained from the [Intel Linux Graphics web
-site](http://intellinuxgraphics.org). If some graphical bugs subsist, changing
-the "RTT prefered mode" from the Webots OpenGL Preferences from "Framebuffer
+Intel may be obtained from the [Intel download center website](http://downloadcenter.intel.com). Linux graphics drivers from Intel may be
+obtained from the [Intel Linux Graphics website](http://intellinuxgraphics.org). If some graphical bugs subsist, changing
+the "RTT preferred mode" from the Webots OpenGL Preferences from "Framebuffer
 Object" to "Pixelbuffer Object" or "Direct Copy" may fix the problems. However,
-this may also impact the 3D performance.
+this may also impact the 3D performances.
```
Comment (omichel) [inline guide/verifying-your-graphics-driver-installation.md]: In English, I believe performance is always singular.

File: guide/webots-built-in-editor.md [hunk 20]
```diff
@@ -16,15 +16,15 @@ highlighting for Webots supported language (C/C++, Java, Python and
...
+The compilation output is printed to the Webots console; errors and warnings appear
 in red. If you double-click an error message, Webots will highlight the
 corresponding source line in the editor.
 
 Note that, for compiling source code it is necessary to have the appropriate
 development tools installed. You will find information on the development tools
 [here](language-setup.md).
 
-In the following, the possible compilation actions are listed. Some of them are
+The possible compilation actions are listed bellow. Some of them are
```
Comment (omichel) [inline guide/webots-built-in-editor.md]: bellow &rarr; below

File: guide/webots-licenses.md [hunk 21]
```diff
@@ -28,8 +28,8 @@ of charge and the users pay only for the modules they need. Various modules are
 available for purchase at different price tags. They include special robot
 models (e.g., NAO, DARwIn-OP, e-puck, Thymio 2, etc.) which can be programmed
 using different programming languages (e.g., C/C++, Matlab, Python) or
-interfaces to third parties software or hardware, etc. They are all described on
-Cyberbotics web site.
+interfaces to third party softwares or hardwares, etc. They are all described on the
```
Comment (omichel) [inline guide/webots-licenses.md]: softwares &rarr; software
Also I believe hardware could be kept singular in this case.

File: guide/webots-licenses.md [hunk 22]
```diff
@@ -43,7 +43,7 @@ EDU and Webots MOD.
 | Supervisor capability                | yes | no  | no         |
 | Physics plug-in programming          | yes | no  | no         |
 | Fast simulation mode                 | yes | no  | no         |
-| Robot and environment modelling      | yes | yes | no         |
+| Robot and environment modeling      | yes | yes | no         |
```
Comment (omichel) [inline guide/webots-licenses.md]: Add some spacing to avoid breaking the columns.

File: guide/the-user-interface.md (hunk 11 above)
Comment (remidhum) [inline guide/the-user-interface.md]: then "some" should be changed to "a", unless it means has the meaning of "in some special cases ..."

Comment (omichel) [inline guide/the-user-interface.md]: Yes, that sounds good.

**Comment by remidhum** [comment]
Note, Hands on and Theory now all start with caps. Removed include-files modifications along with the other typos and suggestions

**Comment by remidhum** [comment]
:+1:

File: guide/using-ros.md [hunk 23]
```diff
@@ -54,10 +54,10 @@ different instances of the same robot, the name of the robot is followed by the
 ID of the process and the IP address of the computer.
 
 "[device\_name]": since the same function can refer to different devices, this
-field show you which device it refers to.
+field shows you which device it refers to.
 
-"[service/topic\_name]": this field is equal or really close to the Webots
-function it corresponds. For topics, it is followed by the sampling period. For
+"[service/topic\_name]": this field is indentical or very close to the Webots
```
Comment (stefaniapedrazzi) [inline guide/using-ros.md]: typo: indentical

File: guide/using-numerical-optimization-methods.md [hunk 24]
```diff
@@ -224,7 +224,7 @@ parameter evaluation. When the robot controller finishes the evaluation, it
 writes the fitness result into another text file and then it calls the
 `wb_supervisor_simulation_quit()` function to terminate Webots. Then the control
 flow returns to the optimization program that can read the resulting fitness,
-associate it with the current genotype and proceed with the next genotype.
+associated it with the current genotype and proceed with the next genotype.
```
Comment (stefaniapedrazzi) [inline guide/using-numerical-optimization-methods.md]: I'm not sure about this modification because the optimization program does three actions: read the fitness, associate the fitness, and proceed.

Maybe if it is not clear we could rephrase the sentence or replace ``it`` with ``fitness``.

File: guide/using-matlab.md [hunk 25]
```diff
@@ -93,10 +92,10 @@ In order to create a cross-platform controller, it is recommended to use the
 We recommend to use the latest MATLAB version on an up-to-date operating system.
 
 Note that 64-bit versions of Webots are not compatible with 32-bit versions of
-MATLAB. Webots comes only in 64-bit flavour and therefore it can only
+MATLAB. Webots comes only in 64-bit flavours and therefore it can only
```
Comment (stefaniapedrazzi) [inline guide/using-matlab.md]: This comment is not related with your change, but could you please change it to the American English version ``flavors``?

File: guide/using-java.md [hunk 26]
```diff
@@ -24,17 +24,17 @@ The Java Development Kit (JDK) can be downloaded for free from the [Sun
 Developer Network](http://www.oracle.com/technetwork/java/javase/downloads).
 Make sure you choose the most recent release and the Standard Edition (SE) of
 the JDK 8. For Windows, make also sure you have selected the 64 bit version
-since webots is incompatible with the 32 bit version. Then follow the
+since webots is not compatible with the 32 bit version. Then, follow the
```
Comment (stefaniapedrazzi) [inline guide/using-java.md]: Another comment not related to your changes that would be convenient to fix in this PR: webots -> Webots

Comment (remidhum) [inline guide/using-java.md]: I'll make a global check for this

File: guide/prerequisites.md [hunk 27]
```diff
@@ -16,17 +16,17 @@ chapter](installing-webots.md).
 The first step is to create a directory which will contain all your files
 related to Webots.
 
-> **handson**:
-From your operating system interface, choose a location on your hard disk where
+> **Hands on**:
+From your operating system  interface, choose a location on your hard disk where
```
Comment (stefaniapedrazzi) [inline guide/prerequisites.md]: it seems that an additional space has been introduced here

File: guide/using-java.md (hunk 26 above)
Comment (stefaniapedrazzi) [inline guide/using-java.md]: thank you

File: guide/using-numerical-optimization-methods.md [hunk 28]
```diff
@@ -224,7 +224,7 @@ parameter evaluation. When the robot controller finishes the evaluation, it
 writes the fitness result into another text file and then it calls the
 `wb_supervisor_simulation_quit()` function to terminate Webots. Then the control
 flow returns to the optimization program that can read the resulting fitness,
-associate it with the current genotype and proceed with the next genotype.
+associate fitness with the current genotype and proceed with the next genotype.
```
Comment (remidhum) [inline guide/using-numerical-optimization-methods.md]: although fitness is repeated I find the sentence makes more sense this way

**Comment by remidhum** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by DavidMansolino** [comment]
(I merged the PR as users who forked the repository can't merge the PR.)
